efficientdet/keras/eval.py

```diff
@@ -56,19 +59,32 @@ def main(_):
   model.build((config.batch_size, 512, 512, 3))
   model.load_weights(FLAGS.checkpoint)
 
-  evaluator = coco_metric.EvaluationMetric(
-      filename=config.val_json_file)
+  evaluator = coco_metric.EvaluationMetric(filename=config.val_json_file)
 
   # compute stats for all batches.
   for images, labels in ds:
-    cls_outputs, box_outputs = model(images, training=False)
     config.nms_configs.max_nms_inputs = anchors.MAX_DETECTION_POINTS
+
+    cls_outputs, box_outputs = model(images, training=False)
     detections = postprocess.generate_detections(config, cls_outputs,
                                                  box_outputs,
                                                  labels['image_scales'],
-                                                 labels['source_ids'])
-    evaluator.update_state(labels['groundtruth_data'].numpy(),
-                           detections.numpy())
+                                                 labels['source_ids'], False)
+
+    images_flipped = tf.image.flip_left_right(images)
```

Comment:
Could you make this as optional (maybe add a flag  like "enable_tta" or "enable_flip")?

Reply:
Done.